doc/sphinxext/sphinx_gallery/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""
+==============
+Sphinx Gallery
+==============
+
+"""
+import os
+__version__ = '0.1.4'
```

Comment:
If the gallery looks fine, you can ignore my comment, merge this PR and update sphinx-gallery in another PR.